src/core/namespace.c

```diff
@@ -1863,6 +1863,7 @@ int setup_namespace(
                                 loop_device->fd,
                                 &verity,
                                 root_image_options,
+                                loop_device->uevent_seqnum_not_before,
```

Comment:
mmh commit message is `sd-device: use right type for usec timestamps` but there's all these changes to function signatures which look like they belong in commit `dissect: ignore old uevents when waiting for loopback partition scan` ?

Reply:
(this is https://github.com/systemd/systemd/pull/19368/commits/3c4a5a85f1e4f6a300e9b3cf76ad0d2682eb5eff )